client/views/admin/integrations/new/NewIntegrationsPage.js

```diff
@@ -45,11 +45,9 @@ export default function NewIntegrationsPage({ ...props }) {
 				{t('Outgoing')}
 			</Tabs.Item>
 		</Tabs>
-		<Page.ScrollableContentWithShadow>
-			{
-				(tab === 'incoming' && <NewIncomingWebhook key='incoming'/>)
-				|| (tab === 'outgoing' && <NewOutgoingWebhook key='outgoing'/>)
-			}
-		</Page.ScrollableContentWithShadow>
+		{
```

Comment:
You need to fix this on the edit page as well, since as of it now, when you click in an integration, it still shows the white page.

Reply:
Sure, I am on it. :)